cypress/integration/elements/LeftNav.js

```diff
@@ -47,13 +47,31 @@ class LeftNav{
         }
     }
 
-    getOpenToWorkspaceButton(){
-        return cy.get('[data-test=open-document-button]');
+    getOpenToWorkspaceButton(index){
+        return cy.get('#leftNavContainer' + index).find('[data-test=open-document-button]');
     }
 
     openToWorkspace(title) {
+        let index = -1;
+        switch(title){
+            case 'Introduction':
+                index = 0;
+                break;
+            case 'Initial Challenge':
+                index = 1;
+                break;
+            case 'What if':
+                index = 2;
+                break;
+            case 'Now What':
+                index = 3;
+                break;
+            case 'Extra Workspace':
+                index = 4;
+                break;
+        }
```

Comment:
@eireland @kevinsantos-cc `openToWorkspace` now finds an index based on the workspace name.  I followed some other code examples where we convert a name to an index using a switch.  Is this ok with folks?  Not sure if this might cause some redundancy or be tricky to update if we add workspaces.